<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Host</string>
   </property>
   <widget class="QLabel" name="labelConnected">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 132, 0);</string>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Send File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>70</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 229, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSocketInfo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 229, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChooseFile">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccepting">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(74, 149, 0);</string>
    </property>
    <property name="text">
     <string>Wainting </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAccept">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
